### PR TITLE
Add probability threshold for classification uploads

### PR DESCRIPTION
**Pull request by bfhealy**
This PR adds a probability threshold argument to scope_upload_classification.py. Only classifications with a probability greater than the threshold will be uploaded. This is useful for active learning purposes and any time we want to post only a subset of potential classifications.

### CODE REVIEW

File: tools/scope_upload_classification.py [hunk 1]
```diff
@@ -82,11 +84,11 @@ def upload_classification(
 
         if read_classes:
             row_classes = row[classes]  # limit current row to specific columns
-            nonzero_keys = row_classes.keys()[
-                row_classes > 0
+            threshold_keys = row_classes.keys()[
+                row_classes > p_threshold
```
Comment (mcoughlin) [inline tools/scope_upload_classification.py:88]: >= ?

File: tools/scope_upload_classification.py [hunk 2]
```diff
@@ -278,6 +280,13 @@ def upload_classification(
 
     parser.add_argument("-ztf_origin", type=str, help="Origin of uploaded ZTF data")
 
+    parser.add_argument(
+        "-p_threshold",
+        type=float,
+        default=0.0,
+        help="Classification probability > this number to upload",
```
Comment (mcoughlin) [inline tools/scope_upload_classification.py:287]: >= ?

**Review by mcoughlin — APPROVED** [review]
I think better is >= rather than >. Otherwise, LGTM!

**Comment by bfhealy** [comment]
@mcoughlin That sounds good - I just made the changes!

**Pull request merged**